app/utils/newsFeed/redis/redisMappings.ts

```diff
@@ -137,3 +154,17 @@ const mapItem = (redisFeedEntry: RedisNewsFeedEntryWithAdditionalData, user: Use
     projectId: projectId,
   }) as NewsFeedEntry['item'];
 };
+
+const updateImageUrl = (imageUrl: string | undefined): string | undefined => {
+  if (imageUrl && imageUrl.startsWith(`${clientConfig.NEXT_PUBLIC_STRAPI_ENDPOINT}`)) {
+    return imageUrl.replace(`${clientConfig.NEXT_PUBLIC_STRAPI_ENDPOINT}`, '');
+  }
+  return imageUrl;
+};
+
+const updateImageUrls = (item: any) => {
+  if (item.author?.image) {
```

Comment:
Not sure if this is necessary, but the type `RedisItem` has a field `followedBy` with users, might need mapping as well

Reply:
forgot to add it here, but the comment about using the `URL` class also applies here